src/app/component/liquidation/LiquidationModal.tsx

```diff
@@ -0,0 +1,103 @@
+import { AiOutlineDollarCircle } from "react-icons/ai";
+import Modal from "../common/Modal";
+import { useState } from "react";
+import LoanSummaryContainer from "../create-loan/LoanSummaryContainer";
+import LoanSummary from "../create-loan/LoanSummary";
+import Button from "../button/Button";
+
+type LiquidationModalProps = {
+  open: boolean;
+  handleClose: () => void;
+};
+
+type CurrencyOption = {
+  name: string;
+  value: string;
+};
+const collateralAmountOptions: CurrencyOption[] = [
+  { name: "USDT", value: "usdt" },
+  { name: "BTC", value: "btc" },
+  { name: "USDC", value: "usdc" },
+  { name: "ETH", value: "eth" },
+  { name: "MATIC", value: "matic" },
+  { name: "SOL", value: "sol" },
+];
+
```

Comment:
Type and constant shall got to constant and types folders,so they can be reused

Reply:
fixed